_episodes/02-image-basics.md

```diff
@@ -67,8 +72,20 @@ real world images will typically be made up of a vast number of pixels,
 and each of these pixels will be one of potentially millions of colours.
 While we will deal with pictures of such complexity shortly,
 let's start our exploration with 15 pixels in a 5 X 3 matrix with 2 colours and
-work our way up to that complexity,
-but first the necessary imports.
+work our way up to that complexitywork our way up to that complexity.
```

Comment:
Sorry, this seems to have gotten doubled in the process.

Reply:
```suggestion
work our way up to that complexity.
```